File: trabajos encargados/trabajo encargado semana2/ejercicios condicionales/ejer9condicional.cpp
#include <iostream>
using namespace std;
int main(){
    cout << "ingrese un dia para una frese del dia" << endl;
    cout << "1: lunes" << endl;
    cout << "2: martes" << endl;
    cout << "3: miercoles" << endl;
    cout << "4: jueves" << endl;
    cout << "5: viernes" << endl;
    cout << "6: sabado" << endl;
    cout << "7: domingo" << endl;
    int opcion;
    cin >> opcion;
    switch (opcion){
        case 1:
        cout << "lunes: " << endl;
        cout << "feliz lunes y que tengas suerte" << endl;
            break;
        case 2:
        cout << "martes: " << endl;
        cout << "este dia tendras buen momento" << endl;
            break;
        case 3:
        cout << "miercoles: " << endl;
        cout << "gran dia ten voluntad" << endl;
            break;
        case 4:
        cout << "jueves: " << endl;
        cout << "un dia bien hermoso y calido" << endl;
            break;
        case 5:
        cout << "viernes: " << endl;
        cout << "este dia tendras mayor calidad de vida" << endl;
            break;
        case 6:
        cout << "sabado: " << endl;
        cout << "este dia es un buen descanso" << endl;
            break;
        case 7:
        cout << "domingo: " << endl;
        cout << "este dia es un buen dia de capacitacion" << endl;
            break;
            default:
        cout << "dia no valida: " << endl;
    }
    return 0;
}

File: trabajos encargados/trabajo encargado semana 1 condicional -- bucles/ejercicios condicionales/ejer9condicional.py
print("Ingrese un dia para una frase del dia")
print("1: Lunes")
print("2: Martes")
print("3: Miercoles")
print("4: Jueves")
print("5: Viernes")
print("6: Sabado")
print("7: Domingo")

opcion = int(input())

if opcion == 1:
    print("Lunes:")
    print("Feliz lunes y que tengas suerte")

elif opcion == 2:
    print("Martes:")
    print("Este dia tendras buen momento")

elif opcion == 3:
    print("Miercoles:")
    print("Gran dia, ten voluntad")

elif opcion == 4:
    print("Jueves:")
    print("Un dia bien hermoso y calido")

elif opcion == 5:
    print("Viernes:")
    print("Este dia tendras mayor calidad de vida")

elif opcion == 6:
    print("Sabado:")
    print("Este dia es un buen descanso")

elif opcion == 7:
    print("Domingo:")
    print("Este dia es un buen dia de capacitacion")

else:
    print("Dia no valido")
